source/Dgraph.tests.e2e/Tests/MutateQueryTest.cs

```diff
@@ -74,66 +69,62 @@ public async override Task Setup()
 
         public async override Task Test()
         {
-            using (var client = await ClientFactory.GetDgraphClient())
-            {
-                await AddThreePeople(client);
-                await QueryAllThreePeople(client);
-                await AlterAPerson(client);
-                await QueryWithVars(client);
-                await DeleteAPerson(client);
-            }
+            using var client = await ClientFactory.GetDgraphClient();
```

Comment:
what is the difference in doing this within a block and outside?

Reply:
Less indentation, and the object is disposed when it leaves the enclosing scope.